var/spack/repos/builtin/packages/blender/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+
+from spack import *
+
+import os
+import shutil
+
+class Blender(Package):
+    """Blender is the free and open source 3D creation suite."""
+
+    homepage = "https://www.blender.org"
+
+    version('2.81a-217', sha256='bb6e03ef79d2d7273336f8cfcd5a3b3f')
+
+    conflicts("python")
+
+    def install(self, spec, prefix):
+	for file in os.listdir(self.stage.source_path):
+            src = os.path.join(self.stage.source_path, file)
+            dst = os.path.join(prefix, file)
+            if os.path.isdir(src):
+                shutil.copytree(src, dst, False)
+            else:
+                shutil.copy2(src, dst)
+
+    def setup_environment(self, spack_env, run_env):
+        blender_python_path = os.path.join(prefix, self.version.string[:4], 'python/lib/python3.7')
+        run_env.set('PYTHONPATH', blender_python_path)
+
+    def url_for_version(self, version):
+	glibc_version = version.string.split('-')[1]
+	blender_version_full = version.string.split('-')[0]
+	blender_version_minor = version.string[:4]
```

Comment:
`version.upto(2)` does not work?

Reply:
Error: AttributeError: 'Version' object has no attribute 'upto'

Reply:
https://spack.readthedocs.io/en/latest/spack.html#spack.version.Version.up_to

Forgot an underscore…

Reply:
It would be better than relying on indices only. What about blender `2.100` or `3.1`?